spec/features/create_etd_spec.rb

```diff
@@ -238,13 +238,13 @@
       expect(page).not_to have_content('up to 7 days')
     end
 
-    # update embargo
+    # Manage embargo
     new_embargo_date = Date.today + 3
     visit "#{Settings.argo_url}/view/#{prefixed_druid}"
-    find_link('Update embargo').click
+    find_link('Manage embargo').click
     within '#blacklight-modal' do
-      fill_in('embargo_date', with: new_embargo_date.strftime('%F'))
-      click_button 'Update Embargo'
+      fill_in('item_embargo_release_date', with: new_embargo_date.strftime('%F'))
```

Comment:
it would be better to use the label text here. "Enter the date when this embargo ends". This helps test the accessability of the form and is easer to read.

Reply:
I'll try that, it didn't work the first time, but I think I may have had something failing before it got there.

Reply:
Done in all 3. Works fine, must have had an earlier failure in the test.